### PR TITLE
Fix video playing when typing on the searh bar

### PR DESCRIPTION
**Pull request by eldiegod**
#318

### CODE REVIEW

File: src/views/viewer/VideoView/VideoView.tsx [hunk 1]
```diff
@@ -34,6 +35,7 @@ export const VideoView: React.FC = () => {
   const { state, updateWatchedVideos } = usePersonalData()
   const timestampFromQuery = Number(useRouterQuery('time'))
   const { getAssetUrl } = useAsset()
+  const searchRouteMatch = !!useMatch({ path: absoluteRoutes.viewer.search() })
```
Comment (kdembler) [inline src/views/viewer/VideoView/VideoView.tsx]: Maybe it'd be safer to have the condition the other way around? So to check whether we are on `VideoView`? Then if we add another overlay, this won't break

Comment (eldiegod) [reply src/views/viewer/VideoView/VideoView.tsx]: good point!

File: src/views/viewer/VideoView/VideoView.tsx [hunk 2]
```diff
@@ -34,7 +35,7 @@ export const VideoView: React.FC = () => {
   const { state, updateWatchedVideos } = usePersonalData()
   const timestampFromQuery = Number(useRouterQuery('time'))
   const { getAssetUrl } = useAsset()
-
+  const searchRouteMatch = useMatch({ path: absoluteRoutes.viewer.video(id) })
```
Comment (kdembler) [inline src/views/viewer/VideoView/VideoView.tsx]: I think we may need to adjust the name of the variable now, sorry :P

Comment (eldiegod) [reply src/views/viewer/VideoView/VideoView.tsx]: ah fk didnt see

**Review by kdembler — APPROVED** [review]

**Pull request merged**